src/wavelengths.jl

```diff
@@ -152,7 +161,10 @@ function Base.Sort.searchsortedfirst(wls::Wavelengths, λ)
     if λ >= 1 # convert Å to cm
         λ *= 1e-8
     end
```

Comment:
I know I'm stating the obvious, but you may want to make an issue to make sure you deal with this before the 1.0 release (or at least better document the behavior)

Reply:
This is intended to be a convenience, since constructing wavelengths doesn't happen often the performance is not a problem.  Thanks for catching that it wasn't documented.